### ISSUE TITLE
Opening sidebar for configuring items in a category doesn't show item list for self-created custom categories.

### ISSUE THREAD

**Issue by Daeadin** [Cidan/BetterBags#674]
### Addon Version

v0.3.10

### World of Warcraft Version

Retail

### Describe the bug

When opening the sidebar for configuring items in a category for self-created custom categories (categories created via typing in the Search bar and hitting Enter), only the 'Configure Search Category' screen appears, which allows you to configure the Category Name, Search Query, and Priority.  No list of items is provided even if items already exist within said custom category.

### Steps to reproduce

Click Better Bags icon upper right of bags, click Configure Categories, click the arrow next to a self-created custom category.

**Comment by AMaribo** [comment]
Have you manually added items to a search category?

**Comment by Daeadin** [comment]
Yes, each of the affected custom categories have items already assigned to
them.

**Comment by Daeadin** [comment]
I should note, each of these categories were created prior to the latest
version of BetterBags and had their items added prior as well.

**Comment by qlesz** [comment]
Also window to create custom category dissapeared from option menu (was gonna make new issue but couldn't somehow).

**Comment by AMaribo** [comment]
> Also window to create custom category dissapeared from option menu (was gonna make new issue but couldn't somehow).

This option has been deprecated.
To create a new custom category, drag an item to the bag button in the top-left corner of the bag.

**Comment by Daeadin** [comment]
Confirmed same additional issue as qlesz describes on my end as well.

**Comment by AMaribo** [comment]
The above error will be fixed later today.

**Comment by AMaribo** [comment]
This is fixed in v0.3.11, heading out to addon sites now.

**Comment by Daeadin** [comment]
Qlesz,

Generally speaking, if you have a new issue you should open a separate bug
report rather than piggyback on an existing one, as this typically allows
the dev to better track and address both issues separately.
